src/index.md

```diff
@@ -102,6 +102,8 @@ description: >
                     emulators, simulators, and hardware for iOS
                     and Android.
                 </p>
+                
+                <a href="development/tools/hot-reload">Learn more</a>
```

Comment:
Did add this line?

Reply:
If not, there might be an issue with the way you updated your branch.

Reply:
No, I did not.

Feeling defeated by the tools.